pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1202,12 +1203,13 @@ func (ctrl *PersistentVolumeController) doDeleteVolume(volume *v1.PersistentVolu
 		// Cannot create deleter
 		return false, fmt.Errorf("Failed to create deleter for volume %q: %v", volume.Name, err)
 	}
-
+	start := time.Now()
 	if err = deleter.Delete(); err != nil {
 		// Deleter failed
+		metrics.VolumeDeleteLatency.Observe(metrics.SinceInMilliseconds(start))
 		return false, err
 	}
-
+	metrics.VolumeDeleteLatency.Observe(metrics.SinceInMilliseconds(start))
```

Comment:
On second thoughts - it may be too much to record volume name as a dimension here. Please ignore what I wrote. :-)